.cirrus.star

```diff
@@ -16,6 +16,7 @@ def main(ctx):
     #   but on the cirrus-ci repo page
     # - commit message containing [wheel build]
     # - a tag that begins with v, but doesn't end with dev0
+    # - all pushes to a maintenance branch
```

Comment:
we still want the `[wheel build]` requirement as Ralf suggests; we can remove the tag trigger above altogether I think (looks like you started doing or have done that in the code below?)

Reply:
tag trigger is removed